### PR TITLE
Openshift-ci deployment to test changes to model-registry against latest public opendatahub image

### PR DESCRIPTION
**Pull request by tonyxrmdavidson**
- target the openshift-ci/script/oci-model-registry-deploy.sh script 
- change the openshift-ci deployment to include a cron job running at 1am 
- add the option to call the script on a PR by adding the label /test e2e-odh-cfo-optional 
- adds configuration to output results into our team slack channel

### CODE REVIEW

**Comment by tonyxrmdavidson** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-cfo-optional

**Comment by tonyxrmdavidson** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-cfo-optional

**Comment by tonyxrmdavidson** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-cfo-optional

**Comment by tonyxrmdavidson** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-cfo-optional

**Comment by tonyxrmdavidson** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-cfo-optional

**Comment by tonyxrmdavidson** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-cfo-optional

**Comment by tonyxrmdavidson** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-cfo-optional

**Comment by tonyxrmdavidson** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-cfo-optional

**Comment by tonyxrmdavidson** [comment]
/pj-rehearse periodic-ci-opendatahub-io-model-registry-main-e2e-odh-cfo-cron

**Comment by tonyxrmdavidson** [comment]
https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/49999/rehearse-49999-pull-ci-opendatahub-io-model-registry-main-e2e-odh-cfo-optional/1774776454932860928

https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/49999/rehearse-49999-periodic-ci-opendatahub-io-model-registry-main-e2e-odh-cfo-cron/1777322112441126912

File: ci-operator/config/opendatahub-io/model-registry/opendatahub-io-model-registry-main.yaml [hunk 1]
```diff
@@ -21,7 +21,8 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: e2e-odh-cfo
+- as: e2e-odh-cfo-cron
```
Comment (rkubis) [inline ci-operator/config/opendatahub-io/model-registry/opendatahub-io-model-registry-main.yaml:24]: I think that `cfo` means Cloudflare Operator here :) We should use `mro` for Model Registry Operator.

File: ci-operator/config/opendatahub-io/model-registry/opendatahub-io-model-registry-main.yaml [hunk 2]
```diff
@@ -32,7 +33,25 @@ tests:
...
       cli: latest
       commands: |
-        echo "Testing for Model-Registry Openshift-CI will happen here."
+        ./openshift-ci/scripts/oci-model-registry-deploy.sh
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    workflow: hypershift-hostedcluster-workflow
+- as: e2e-odh-cfo-optional
```
Comment (rkubis) [inline ci-operator/config/opendatahub-io/model-registry/opendatahub-io-model-registry-main.yaml:42]: The same as above :) Change `cfo` to `mro`.

**Review by rkubis — CHANGES_REQUESTED** [review]
Please just change `cfo` to `mro` everywhere to properly recognize product/component :) Thanks!

**Comment by tonyxrmdavidson** [comment]
This PR was pulled, updated and pushed as part of another PR

**Comment by tonyxrmdavidson** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-mro-optional

**Comment by rkubis** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-images pull-ci-opendatahub-io-model-registry-main-e2e-odh-mro-optional

**Comment by tonyxrmdavidson** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-mro-optional

**Comment by tonyxrmdavidson** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-mro-optional

**Comment by tonyxrmdavidson** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-mro-optional

**Comment by tonyxrmdavidson** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-images pull-ci-opendatahub-io-model-registry-main-e2e-odh-mro-optional

**Comment by tonyxrmdavidson** [comment]
forum-ocp-testplatform team notified of catalogsource failure. Jira [here](https://issues.redhat.com/browse/DPTP-3956)

**Comment by bear-redhat** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-mro-optional

**Comment by tonyxrmdavidson** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-mro-optional

**Comment by tonyxrmdavidson** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-mro-optional

**Comment by tonyxrmdavidson** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-mro-optional

**Comment by tonyxrmdavidson** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-mro-optional

**Comment by tonyxrmdavidson** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-mro-optional

**Comment by tonyxrmdavidson** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-images

**Comment by tonyxrmdavidson** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-mro-optional

**Comment by tonyxrmdavidson** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-images

**Comment by tonyxrmdavidson** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-mro-optional

**Comment by rkubis** [comment]
/pj-rehearse pull-ci-opendatahub-io-model-registry-main-e2e-odh-mro-optional

**Pull request merged**